### PR TITLE
Markerandroid

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: shared/components/map.js [hunk 1]
```diff
@@ -321,10 +319,10 @@ export default class Map extends Component {
               this.props.onMarkerClick(item);
               this.cardAnimationSlideUp();
               this.buttonAnimationSlideUp();
-              this.setState({markerSelect: item.key});
+              this.state.markerSelect = item.key;
```
Comment (yuda1124) [inline shared/components/map.js]: use set State! why did you change?

File: android/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <resources>
 
-    <string name="app_name">Goober</string>
+    <string name="app_name">Pingo</string>
```
Comment (yuda1124) [inline android/app/src/main/res/values/strings.xml]: good work!

File: shared/components/map.js [hunk 3]
```diff
@@ -324,7 +322,7 @@ export default class Map extends Component {
               this.setState({markerSelect: item.key});
             }}
           >
-            {(this.state.markerSelect === item.key) ?
+            {(Platform.OS === 'ios' && this.state.markerSelect === item.key) ?
```
Comment (yuda1124) [inline shared/components/map.js]: If we have to implement android case, plz write to do! There are some problem in android?

Comment (ochanje210) [inline shared/components/map.js]: yes. rendering any components inside of MapView.Marker results fatal app crash.

Comment (ochanje210) [inline shared/components/map.js]: so until figuring out this issue, it would be hard to render numbers on clicked marker
